src/data/attribute-dictionary.json

```diff
@@ -12196,6 +12196,73 @@
     "definition": "<p>NrAiSignal shows details from every NRQL alert condition and every signal on your account, for every aggregation window that passes. This data is posted immediately after each aggregation window is aggregated and evaluated, so it will show you exactly what <a href=\"https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page\">our alerts</a> are seeing.</p>\n",
     "name": "NrAiSignal"
   },
+  {
+    "attributes": [
+      {
+        "definition": "<p>Name of the destination used to authenticate this request.</p>\n",
```

Comment:
Authenticate is not exactly accurate, the destination is the whole connection between our system with the 3rd party. Personally I'd say Name of the destination used to send the notification